modules/SFM/SFM.cpp

```diff
@@ -411,9 +417,19 @@ bool SFM::updateModule()
             
     if (outDisp.getOutputCount()>0)
     {
-        outputD=stereo->getDisparity();
+        outputD = stereo->getDisparity();
         ImageOf<PixelMono> &outim=outDisp.prepare();
-        outim.wrapIplImage(&outputD);
+        if (doBLF)
+        {
+            IplImage* outputDpt = &outputD;
```

Comment:
@tanismar don't you need a `cvReleaseImage(&outputDpt);` when you have finished with this?